### ISSUE TITLE
Text Selection Forced when using LineEdit/TextEdit

### ISSUE THREAD

**Issue by rokasv** [godotengine/godot#31183]
Godot 3.1.1

I am interested in implementing a custom text input field.
To do this, I have 2 options:
1. Use the functionality of LineEdit TextEdit and hide the styling
2. Read input manually

If I go with LineEdit, I can update the text, however I am forced to support visuals for selections. Because [It features many built-in shortcuts which will always be available](https://docs.godotengine.org/en/3.1/classes/class_lineedit.html#description). 

In other words, I have no visuals for text selection and do not want to support it. However disabling this feature is not possible using LineEdit/TextEdit nodes.

Going with the other option gimps me in terms of supporting unicode characters, decoding which from scancode/unicode symbols is not doable (ąčęėįš codes do not convert into the appropriate string letters using ` PoolByteArray([event.unicode]).get_string_from_utf8()` magic from [here](https://github.com/godotengine/godot/issues/6166)).

**TL;DR: please allow disabling text selection and/or shortcuts available when editing text.**
